### DISCUSSION TITLE
Adding HTML to Cells in AG-grid

### DISCUSSION THREAD

**Discussion by JS-Aibel** [zauberzeug/nicegui#423] [Q&A] [answered]
In JustPy one can add html directly in a cell as shown below. Can this be implemented in NiceGUI as well?
Or is it any other way to get a clickable link in a AG-grid cell with NiceGUI?

**Example**:
```
    'rowData': [
      {'name': "NiceGUI", 'link': '<a href="http://nicegui.io"> NiceGUI website </a>'}
    ]

grid.html_columns = [1]
```

https://justpy.io/grids_tutorial/adding_html/

**Comment by falkoschindler** [comment, marked as answer]
Yes, there is a `html_columns` parameter for specifying columns with HTML content.

Here is an example from a recent [Reddit post](https://www.reddit.com/r/nicegui/comments/10zsy92/link_from_within_a_certain_table_cell/):

```py
ui.table({
    'columnDefs': [{'field': 'name'}, {'field': 'phone'}],
    'rowData': [{'name': 'Alice', 'phone': '<a href="tel:+12025550132">+1-202-555-0132</a>'}],
}, html_columns=[1])
```

See https://github.com/zauberzeug/nicegui/issues/217#issuecomment-1370917999 for another example.
